apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/StorageClientImpl.cs

```diff
@@ -67,13 +69,18 @@ public static string ApplicationName
         /// <inheritdoc />
         public override StorageService Service { get; }
 
+        /// <inheritdoc />
+        public override EncryptionKey EncryptionKey { get; }
+
         /// <summary>
         /// Constructs a new client wrapping the given <see cref="StorageService"/>.
         /// </summary>
         /// <param name="service">The service to wrap. Must not be null.</param>
-        public StorageClientImpl(StorageService service)
+        /// <param name="encryptionKey">Optional <see cref="EncryptionKey"/> to use for all object-based operations by default. May be null.</param>
+        public StorageClientImpl(StorageService service, EncryptionKey encryptionKey = null)
         {            
             Service = GaxPreconditions.CheckNotNull(service, nameof(service));
+            EncryptionKey = encryptionKey;
```

Comment:
What do you think about having this fallback to `EncryptionKey.None` so the `EncryptionKey` property is always non-null?